.gitignore

```diff
@@ -103,4 +103,7 @@ local.properties
 .texlipse
 .idea/
 
-/testnet
\ No newline at end of file
+/testnet
+*.cmake
```

Comment:
This is bad. There are cmake files that are under version control and need to be.

Reply:
alright. my bad. I will remove it.

Reply:
fixed.